mantidproject/mantid
# Add notebook back to macOS bundle

**Description of work.**

Moving to Python 3 accidentally removed the notebook functionality from the macOS bundle. This adds it back in.

It also shifts the management of the Python libraries in the bundle to a requirements file for easier updating. 

**To test:**

* Build package on macOS.
* Start Workbench & MantidPlot to check they wor correctly.
* Try the instructions at https://docs.mantidproject.org/nightly/concepts/PythonNotebook.html?highlight=notebook for macOS to check it starts a Jupyter notebook server.


Fixes #28934 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
